examples/Cargo.toml

```diff
@@ -11,6 +11,7 @@ path = "../"
 
 [dependencies]
 num = "^0.1"
+libc = "*"
```

Comment:
i'll change it but, won't this take the last version of libc? should that be a problem (it SHOULD work with the last version of libc)